### ISSUE TITLE
Error when running

### ISSUE THREAD

**Issue by matt2005** [pschmitt/docker-hass-slave#1]
I get the follwoing error when running the container.
AttributeError: module 'homeassistant.remote' has no attribute 'HomeAssistant'

**Comment by pschmitt** [comment]
Yeah the whole HASS slave aspect has pretty much been deprecated.
